### PR TITLE
Adding start, end and run time for backup

### PR DESCRIPTION
**Pull request by vn0bbin**
This will help to check how much time it took to complete the backup.

### CODE REVIEW

**Comment by vn0bbin** [comment]
@djdefi can you please let me know what's the issue here?

**Comment by djdefi** [comment]
Hey there @vn0bbin thanks for the contribution! I've requested a review from the team responsible for `backup-utils` currently.

It looks like the Lint code base check is complaining about some existing `shellcheck` messages, which are unrelated to your contribution. This is not a required check, so I don't think there is any action needed on your part.

**Comment by vn0bbin** [comment]
Got it, Thanks much!

File: bin/ghe-backup [hunk 1]
```diff
@@ -139,6 +139,9 @@ fi
 
 echo "$GHE_SNAPSHOT_TIMESTAMP $$" > ../in-progress
 
+START_TIME=$(date +%s)
+echo 'Start time:' $START_TIME
+
```
Comment (donal) [inline bin/ghe-backup:144]: Remove this empty line.

File: bin/ghe-backup [hunk 2]
```diff
@@ -270,5 +273,9 @@ fi
 echo "Checking for leaked ssh keys ..."
 ghe-detect-leaked-ssh-keys -s "$GHE_SNAPSHOT_DIR" || true
 
+END_TIME=$(date +%s)
+echo 'End time:' $END_TIME
+echo 'Runtime:' $(($END_TIME - $START_TIME)) 'seconds'
+
```
Comment (donal) [inline bin/ghe-backup:279]: These 4 lines should go above line 257: `echo "Completed backup of $GHE_HOSTNAME in snapshot $GHE_SNAPSHOT_TIMESTAMP at $(date +"%H:%M:%S")`

**Review by donal — COMMENTED** [review]
@vn0bbin Thanks for submitting this — I've made a couple of minor suggestions.

File: bin/ghe-backup (hunk 2 above)
Comment (vn0bbin) [reply bin/ghe-backup]: Made the suggested changes, please review.

**Comment by vn0bbin** [comment]
@donal Thanks for the suggestions. Made the required changes, please review it and let me know. Thanks

**Review by donal — APPROVED** [review]

**Comment by vn0bbin** [comment]
@donal Is there anything else required to get this PR merged?

**Comment by vn0bbin** [comment]
@donal Anything else needed to merge this PR?

**Review by gamefiend — APPROVED** [review]
Sorry for the delay on this, approved.

Thanks for your contribution!

**Pull request merged**